website/docs/tutorials/06-deploying/02-migrations/call.md

```diff
@@ -0,0 +1,50 @@
+# `call`
+
+```python
+def call(
+    contract_address: int,
+    function_name: str,
+    inputs: Optional[Union[List[int], Dict[str, Any]]] = None,
```

Comment:
Could we use modern Python's syntax for typing here? I think it would be much more readable:

```suggestion
    inputs: list[int] | dict[str, Any] | None = None,
```